### PR TITLE
MODIN: fix getting started samples testing in CI

### PR DESCRIPTION
**Pull request by dchigarev**
Signed-off-by: Dmitry Chigarev <dmitry.chigarev@intel.com>

# Existing Sample Changes
## Description

This PR is supposed to fix [failing CI testing](https://github.com/oneapi-src/oneAPI-samples/pull/843#commitcomment-66044784) for Modin's getting started samples. The CI flow has been failing due to conda's failures during environment initialization inside the bash script (conda/conda#7980). The CI script was modified as follows to resolve the problem:
1. Source `conda.sh` before using conda to be able to activate environments inside a bash script (conda/conda#7980).
2. Use [`set -e`](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html#:~:text=%2De,function%20call%20completes.) at the beginning of the CI script to ease debugging if something goes wrong.
3. Install notebook's dependencies from [`requirements.txt`](https://github.com/oneapi-src/oneAPI-samples/blob/master/AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/requirements.txt) rather than manually listing them.

Fixes [Jira REIE-1371](https://jira.devtools.intel.com/browse/REIE-1371).

## External Dependencies

No external dependencies were added.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- [x] Command Line

To test the changes locally:
1. Combine steps from the `sample.json` into a single bash script.
2. Run the script and ensure that the last notebook's cell printed"[CODE_SAMPLE_COMPLETED_SUCCESFULLY]"

### CODE REVIEW

**Review by praveenkk123 — APPROVED** [review]
Approve for CI

**Comment by dchigarev** [comment]
For some reason, the CI container still can't import `modin` package during the notebook's flow. I guess this is because activating conda environment and the notebook execution were split into different cells (`env` and `steps` in `sample.json`). I've combined them together to see if it helps, @praveenkk123  could you rerun CI, please?

File: AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/sample.json [hunk 1]
```diff
@@ -10,11 +10,17 @@
   "ciTests": {
   	"linux": [
     {
-  		"env": ["source /opt/intel/oneapi/setvars.sh --force", "conda create -n aikit-modin-test -c intel -c conda-forge matplotlib runipy intel-aikit-modin", "source activate aikit-modin-test"],
+  		"env": [],
```
Comment (dchigarev) [inline AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/sample.json:13]: For some reason, placing environments setup and the notebook execution in different cells (`env` and `steps`) caused the environment wasn't applied (this was indicated by the "Import error: couldn't find 'modin'" error at CI logs).

Placing environment setup steps and the actual notebook execution in a single cell resolved the problem.

**Comment by dchigarev** [comment]
@praveenkk123 thanks for rerunning! CI is green now

**Pull request merged**